src/lib/journey/journey.store.ts

```diff
@@ -54,10 +54,25 @@ export function initialize(initOptions?: StepOptions): JourneyStore {
     });
 
     try {
-      /**
-       * Initial attempt to retrieve next step
-       */
-      nextStep = await FRAuth.next(prevStep as FRStep, options);
+      if (resumeUrl) {
+        // If resuming an unknown journey remove the tree from the options
+        options.tree = undefined;
+
+        /**
+         * Attempt to resume journey
+         */
+        nextStep = await FRAuth.resume(resumeUrl, options);
+      } else if (prevStep) {
+        // If continuing on a tree remove it from the options
+        options.tree = undefined;
```

Comment:
Also discovered in my testing of Transactional Authorization (SDK, not Widget) that journey continuation works best without `authIndex*` parameters. So, removing this if we have a previous step to prevent bugginess.

Reply:
Makes sense now I understand this a bit better